apps/clamav.sh

```diff
@@ -36,7 +36,9 @@ else
     rm -f /etc/systemd/system/clamav-daemon.service
     rm -f "$SCRIPTS"/clamav-fullscan.sh
     rm -f "$VMLOGS"/clamav-fullscan.log
+    rm -f "$SCRIPTS/nextcloud-av-notification.sh"
     crontab -u root -l | grep -v 'clamav-fullscan.sh'  | crontab -u root -
```

Comment:
It should still delete any old entry if this script was run previous to this change imo.